src/document/sidebar.ml

```diff
@@ -1,9 +1,18 @@
 open Odoc_utils
 open Types
+module Id = Odoc_model.Paths.Identifier
 
 let sidebar_toc_entry id content =
-  let href = id |> Url.Path.from_identifier |> Url.from_path in
-  let target = Target.Internal (Resolved href) in
+  let target =
+    match
+      (id :> Odoc_model.Paths.Identifier.t)
+      |> Url.from_identifier ~stop_before:false
+    with
```

Comment:
What are we doing with aliases in the sidebar?

Reply:
@panglesd what do you think about it? It looks to me that we don't lose anything from the previous implementation, that's something that could be improved in a future PR.

Reply:
I don't know how much you have taken from the original PR, but in the original PR we keep track of which module have an expansion.
If it has one, we use `~stop_before:false`, if it has none we use `stop_before:true`. So, aliases appear in the sidebar.

Reply:
That's good, we need to keep that behaviour!

Reply:
The `has_expansion` bool is recorded [here](https://github.com/ocaml/odoc/pull/1220/files#diff-1304c513720e9777b80d9e0c31d217f5b41a5e025b50f7fbd996a534c4ef3c15R41) and used [here](https://github.com/ocaml/odoc/pull/1220/files#diff-d44f4d07a7ce1288d4c91f2a6a38230bb446b85e91fa5d2536f93a3da13eaa0dR108).

I think there is some refactoring to do though, in the final version: `sidebar_toc_entry` is [still there](https://github.com/ocaml/odoc/pull/1220/files#diff-d44f4d07a7ce1288d4c91f2a6a38230bb446b85e91fa5d2536f93a3da13eaa0dR4), but [used only for _pages_ sidebar](https://github.com/ocaml/odoc/pull/1220/files#diff-d44f4d07a7ce1288d4c91f2a6a38230bb446b85e91fa5d2536f93a3da13eaa0dR45). This should be made explicit!

Reply:
> I don't know how much you have taken from the original PR, but in the original PR we keep track of which module have an expansion.

I've checked: currently, what has been taken is mostly refactoring: introducing a tree type and using it in each tree structure, and moving things in the right directory. So, in the current state of this PR the sidebar is only showing root modules (as before), so no need to handle aliases!